scripts/quick_script.sh

```diff
@@ -0,0 +1,9 @@
+mev_inspect_pool_id=$(kubectl get pods | sed -n -e '/^mev-inspect-/p' | sed '/^mev-inspect-workers/d' | awk '{print $1}')
```

Comment:
```suggestion
#!/bin/bash

# This script will run the profit analysis using the specified number of files containing the USD profits - the CSV ones.
# Then, it will copy the analysis results from the Kubernetes cluster to the host machine.

mev_inspect_pool_id=$(kubectl get pods | sed -n -e '/^mev-inspect-/p' | sed '/^mev-inspect-workers/d' | awk '{print $1}')
```